File: GFG/Arrays/stockbs.py
def stocks(arr):
    pro=0
    n=len(arr)
    for i in range(1,n):
        if (arr[i]>arr[i-1]):
            pro+= arr[i]-arr[i-1]
    return pro

brr=[1,5,3,8,12]
print(stocks(brr))
            
            

File: GFG/Arrays/stockbs.cpp
#include<iostream>
#include<vector>
using namespace std;

int stocks(vector<int> &arr){
    int pro=0;
     int n=arr.size();
    for(int i=1;i<n;i++){
        if(arr[i]>arr[i-1]){
            pro=pro+(arr[i]-arr[i-1]);
        }
    }
    return pro;
}
int main(){
    vector<int> arr={1,5,3,8,12};
    cout<<stocks(arr);
    return 0;
}